cfme/rest/__init__.py

```diff
@@ -0,0 +1,28 @@
+import fauxfactory
+from utils.wait import wait_for
+
+
+def service_catalogs(request, rest_api):
```

Comment:
The name of this function is not immediately clear what it does. I see we are adding some catalogs, seems like this should be called ```add_random_service_catalogs``` or something? Some comments too on use etc?